src/rating/Rating.js

```diff
@@ -69,28 +69,40 @@ export default class Rating extends Component {
         this.setState({ value: gesture.dx });
       },
       onPanResponderRelease: event => {
-        onFinishRating(this.getCurrentRating());
+        const rating = this.getCurrentRating();
+        if (!fractions) {
+          // "round up" to the nearest star/rocket/whatever
+          this.setCurrentRating(rating);
```

Comment:
I've also tweaked the image rendering so that if `fractions` is 0 or undefined, individual images will "fill up" to match the rounding up of rating values. Setting `fractions` to a value between 0 and 20 disables this behaviour.